HW11-12_decorators/3_decorator_multiple_print.py

```diff
@@ -0,0 +1,29 @@
+# writing a decorator - printing the name of the function and time of its call
+
+from datetime import datetime
+
+def my_decorator(func):
+    print("Let's wrap our function")
+    print()
+
+    def wrapper(*args):
+        for x in range(times):
+            time = datetime.now()
+            func(*args)
+            print(f"The function name is {func}")
+            print(f"The time of function call is {time}")
+            print()
+    return wrapper
+
+@my_decorator
```

Comment:
there is the task to add parameter to the decorator like:
```suggestion
@my_decorator(times=2)
```